portfolio/src/main/webapp/style.css

```diff
@@ -1,10 +1,92 @@
 body {
     margin: 0px;
+    background-color: white;
 }
 
 #title {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+    font-weight: bold;
+    margin-left: auto;
+    margin-right: auto;
+    width: 100%;
+    background-color: #9fa2b2;
+    height: 100%;
+    padding: 10px;
+    text-align: center;
```

Comment:
Per [style guide](https://google.github.io/styleguide/htmlcssguide.html#Declaration_Order) can you alphabetize all of the properties within each style?